cli/init/templates/tt.yaml.default

```diff
@@ -0,0 +1,64 @@
+tt:
+  modules:
+    # Directory where the external modules are stored.
+    directory: {{ if (eq .CliConfig.Modules.Directory "") }}""{{ else }}{{ .CliConfig.Modules.Directory }}{{ end }}
+
+  app:
+    # Directory that stores various instance runtime
+    # artifacts like console socket, PID file, etc.
+    run_dir: {{ if (eq .CliConfig.App.RunDir "") }}""{{ else }}{{ .CliConfig.App.RunDir }}{{ end }}
+
+    # Directory that stores log files.
+    log_dir: {{ if (eq .CliConfig.App.LogDir "") }}""{{ else }}{{ .CliConfig.App.LogDir }}{{ end }}
+
+    # The maximum size in MB of the log file before it gets rotated.
+    log_maxsize: {{ .CliConfig.App.LogMaxSize }}
+
+    # The maximum number of days to retain old log files.
+    log_maxage: {{ .CliConfig.App.LogMaxAge }}
+
+    # The maximum number of old log files to retain.
+    log_maxbackups: {{ .CliConfig.App.LogMaxBackups }}
+
+    # Restart instance on failure.
+    restart_on_failure: {{ .CliConfig.App.Restartable }}
+
+    # Directory where write-ahead log (.xlog) files are stored.
+    wal_dir: {{ if (eq .CliConfig.App.WalDir "") }}""{{ else }}{{ .CliConfig.App.WalDir }}{{ end }}
```

Comment:
Why cannot we use the following:
`"{{ .CliConfig.App.WalDir }}" `
?

Reply:
This construction is needed in case we have an empty value of WalDir. When using your proposed construction and having the empty string as value, we receive the next content:
```yaml
   wal_dir:
```
which does not correspond to correct yaml format AFAIK.

Reply:
Sure, you are right, fixed.